extensions/infinispan-client/runtime/src/main/java/io/quarkus/infinispan/client/runtime/InfinispanClientProducer.java

```diff
@@ -48,6 +48,7 @@
     private RemoteCacheManager cacheManager;
     @Inject
     private BeanManager beanManager;
+    private InfinispanClientConfigRuntime infinispanClientConfigRuntime;
```

Comment:
I don't understand how it's supposed to work. This is not used anywhere.

Reply:
the explanation above 🙈